### ISSUE TITLE
Get loss to perform backward()

### ISSUE THREAD

**Issue by JonathanAndradeSilva** [open-mmlab/mmdetection#5295]
Hi!

I'm trying to adapt the GRAD-CAM pytorch library to work with mmdetection. My question is:

1) A step is obtain the loss and compute the backward (loss.backward()). In this case, the only possibility is using the forward 
function setting returnLoss==True and setting the gt_boxes and gt_labels?

2) If it is the only option, the backward only obtain the gradient and do not update the weights or the loss.backward() update the wieghts with regards to the annotation info?

Thanks!

**Comment by ZwwWayne** [comment]
1. Yes.
2. loss.backwards update the weights. Please read the pytorch documentation carefully.
